docs/network-flow-visibility.md

```diff
@@ -819,11 +826,120 @@ storage size, please modify the `sizeLimit` in the following section.
   name: clickhouse-storage-volume
 ```
 
+To deploy ClickHouse with Persistent Volumes and limited storage size, please refer
+to [Persistent Volumes](#persistent-volumes).
+
 The time interval between the batch commits to the ClickHouse is specified in the
 [Flow Aggregator Configuration](#configuration-1) as `commitInterval`. The
 ClickHouse throughput grows sightly when the commit interval grows from 1s to 8s.
 A commit interval larger than 8s provides little improvement on the throughput.
 
+##### Persistent Volumes
+
+By default, ClickHouse is deployed in memory. From Antrea v1.7, the ClickHouse
+supports deployment with Persistent Volumes.
```

Comment:
s/the ClickHouse supports deployment with Persistent Volumes/we support deploying ClickHouse with Persistent Volumes

Reply:
Updated
same below